src/App.test.js

```diff
@@ -4,6 +4,9 @@ import App from './App';
 
 it('renders without crashing', () => {
   const div = document.createElement('div');
+  const head = document.createElement('head');
   ReactDOM.render(<App />, div);
+
+  ReactDOM.render(<App />, head);
```

Comment:
**chore** 🧹
Please add a test that checks the text has changed

<sub><sup>I'm using [conventional comments](https://conventionalcomments.org/) formatting to improve comments readability and reader's expectations in this PR.</sup></sub>

Reply:
I already tested it in the previous PR.